### ISSUE TITLE
Viewing EXPLAIN results has word-wrapped output

### ISSUE THREAD

**Issue by robd003** [crate/crate-admin#838]
When viewing `EXPLAIN` results they are crammed into a `max-width: 400px` cell.

It would be great to have unbounded cells for results with a single column.

**Comment by surister** [comment]
As crate-admin is no longer been developed, it might make more sense to feature request this in the new [admin ui](https://github.com/crate/crate-gc-admin) cc @SStorm

**Comment by SStorm** [comment]
Yes thank you. There is a plan to completely rework how explain plans are done to make it more user friendly.

**Comment by proddata** [comment]
> When viewing `EXPLAIN` results they are crammed into a `max-width: 400px` cell.
> 
> It would be great to have unbounded cells for results with a single column.

If you disable formatting ("Format results") then you get the full output:
<img width="1208" alt="image" src="https://github.com/crate/crate-admin/assets/23557193/75fb6f05-150a-4713-a532-cc2b3fcec7f0">

**Comment by robd003** [comment]
Thanks for that @proddata
